server/resolvers/TestPlanReport/issuesResolver.js

```diff
@@ -1,14 +1,14 @@
 const { GithubService } = require('../../services');
 const { Base64 } = require('js-base64');
-const moment = require('moment');
+const dayjs = require('dayjs');
 
 const issuesResolver = async testPlanReport => {
     if (!testPlanReport.candidateStatusReachedAt) return [];
 
     const searchPrefix = `${testPlanReport.at.name} Feedback: "`;
     const searchTestPlanVersionTitle =
         testPlanReport.testPlanVersion.dataValues.title;
-    const searchTestPlanVersionDate = moment(
+    const searchTestPlanVersionDate = dayjs(
```

Comment:
Non-blocking: Would it be better to use one of the util functions instead of `dayjs` directly?

Reply:
Which util functions are you referring to there?

Reply:
Looks like this is the same as `convertDateToString`